sqlalchemy_history/transaction.py

```diff
@@ -16,7 +16,7 @@ def compile_big_integer(element, compiler, **kw):
 
 
 class TransactionBase(object):
-    issued_at = sa.Column(sa.DateTime, default=lambda: datetime.datetime.now(datetime.timezone.utc))
+    issued_at = sa.Column(sa.DateTime(timezone=True), default=lambda: datetime.datetime.now(datetime.timezone.utc))
```

Comment:
Hi @DoingCG 

Thanks for this PR, 
can you also add a testcase corresponding to the change and update documentation mentioning the change in the PR itself, will try to include this in next release.

Thanks